src/openforms/js/components/admin/form_design/logic/actions/dmn/DMNActionConfig.js

```diff
@@ -0,0 +1,232 @@
+
+  useAsync(async () => {
+    if (!pluginId || !decisionDefinitionId) {
+      setFieldValue('decisionDefinitionVersion', '');
+      setDecisionDefinitionVersions([]);
+      return;
+    }
+
+    const response = await get(DMN_DECISION_DEFINITIONS_VERSIONS_LIST, {
+      engine: pluginId,
+      definition: decisionDefinitionId,
+    });
+
+    const versionChoices = response.data.map(item => [item.id, item.label]);
+    setDecisionDefinitionVersions(versionChoices);
+
+    // If decisionDefinitionId has changed, reset the value of the definition version if the new decision definition
+    // does not have that version
+    if (
+      values.decisionDefinitionVersion !== '' &&
+      !versionChoices.find(item => item[0] === values.decisionDefinitionVersion)
+    ) {
+      setFieldValue('decisionDefinitionVersion', '');
+    }
+  }, [pluginId, decisionDefinitionId, setFieldValue]);
+
+  return (
+    <Field
+      name="decisionDefinitionVersion"
+      htmlFor="decisionDefinitionVersion"
+      label={
+        <FormattedMessage
+          defaultMessage="Decision definition version"
+          description="Decision definition version label"
+        />
+      }
+    >
+      <Select
+        id="decisionDefinitionVersion"
+        name="decisionDefinitionVersion"
+        allowBlank={true}
+        choices={decisionDefinitionVersions}
+        {...getFieldProps('decisionDefinitionVersion')}
+      />
+    </Field>
+  );
+};
+
+const DMNActionConfig = ({initialValues, onSave}) => {
+  const {formVariables, plugins} = useContext(FormContext);
+
+  const validate = values => {
+    const errors = {};
+
+    if (values.pluginId === '') {
+      errors.pluginId = 'required';
+    }
+
+    if (values.decisionDefinitionId === '') {
+      errors.decisionDefinitionId = 'required';
+    }
+
+    return errors;
+  };
+
+  return (
+    <div className="dmn-action-config">
+      <Formik initialValues={initialValues} onSubmit={values => onSave(values)} validate={validate}>
+        {formik => (
+          <Form>
+            <fieldset className="aligned">
+              <div className="form-row">
+                <Field
+                  name="pluginId"
+                  htmlFor="pluginId"
+                  label={
+                    <FormattedMessage defaultMessage="Plugin ID" description="Plugin ID label" />
+                  }
+                  errors={
+                    formik.touched.pluginId && formik.errors.pluginId
+                      ? [ERRORS[formik.errors.pluginId]]
+                      : []
+                  }
+                >
+                  <Select
+                    id="pluginId"
+                    name="pluginId"
+                    allowBlank={true}
+                    choices={plugins.availableDMNPlugins.map(choice => [choice.id, choice.label])}
+                    {...formik.getFieldProps('pluginId')}
+                  />
+                </Field>
```

Comment:
can we make this a component that automatically selects the only option if there's only one? :) You can then also make the field readonly

Reply:
Added this to the 'nice to have'